compose.py

```diff
@@ -315,9 +315,10 @@ def create_argmap(images):
         'Use specified Docker image to generate Docker image. Specified as <image-name>,<full-image-name>. <image-name> can be "min" or "full". Both "min" and "full" need to be specified at the same time. This will override "--container-version".'
     )
     parser.add_argument('--enable-gpu',
-                        action="store_true",
-                        required=False,
-                        help='Generate a Triton image that supports GPU.')
+                        nargs='?',
+                        type=lambda x: (str(x).lower() == 'true'),
+                        const=True,
+                        required=True)
```

Comment:
This flag doesn't need to be required, does it?

Reply:
I thought the whole point of keeping --enable-gpu is to keep backward compatibility